Aggiunti Esercizi

pavva91 opened the pull request:
aggiunte soluzioni esercizi eserciziario

Commit:
es 16 add

Commit:
Aggiunta esercizi Valerio

Commit:
Esercizi Valerio

ilario-pierbattista commented:
Valè, se guardi i comparatori di file ti accorgerai che le tue modifiche aggiungono un carattere strano alla fine di ogni riga. È il newline di windows, secondo me, vedi se riesci a toglierlo.
Aspetto nuove modifiche

pavva91 commented:
Ok, non avevo visto, vedo di sistemarlo. ;)

Il giorno 24 gennaio 2017 18:27, Ilario Pierbattista <
notifications@github.com> ha scritto:

> Valè, se guardi i comparatori di file ti accorgerai che le tue modifiche
> aggiungono un carattere strano alla fine di ogni riga. È il newline di
> windows, secondo me, vedi se riesci a toglierlo.
> Aspetto nuove modifiche
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ilario-pierbattista/prolog-utils/pull/4#issuecomment-274875001>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI8_7aKYHPmxRMNQUq59rjZVIhGLSONJks5rVjSGgaJpZM4Lsc7z>
> .
>

Commit:
Create .gitattributes

creato file secondo guida  https://help.github.com/articles/dealing-with-line-endings/

Commit:
Normalize all the line endings

pavva91 commented:
Ciao Ila! Ora mi sembra ok

ilario-pierbattista commented:
Ottimo! Grazie!
